src/GlobalCoders.PSP.BackendApi/Data/Configurations/SurchargeEntityConfiguration.cs

```diff
@@ -0,0 +1,58 @@
+using GlobalCoders.PSP.BackendApi.SurchargeManagement.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace GlobalCoders.PSP.BackendApi.Data.Configurations;
+
+public class SurchargeEntityConfiguration : IEntityTypeConfiguration<SurchargeEntity>
+{
+    public void Configure(EntityTypeBuilder<SurchargeEntity> builder)
+    {
+        builder.ToTable("Surcharges");

```

Comment:
Can you remove everithing from 11 line to 55?
All of this will be generated via migration automaticaly.
And fields like required should be set up, with [Required] attribute in Entity class

In this configuration we are writing only something very custom - like query filter or defining custom relations via tables